src/tools/clippy/tests/ui/wildcard_enum_match_arm.rs

```diff
@@ -7,7 +7,8 @@
     dead_code,
     clippy::single_match,
     clippy::wildcard_in_or_patterns,
-    clippy::unnested_or_patterns
+    clippy::unnested_or_patterns,
+    clippy::diverging_sub_expression
```

Comment:
```suggestion
    clippy::unnested_or_patterns, clippy::diverging_sub_expression
```

Also leaving this comment here, so you could use GitHubs UI to make this change, if you want.